swapper/swapper.py

```diff
@@ -106,3 +108,101 @@ def copy_and_replace(fc):
                 arcpy.management.ChangePrivileges(renamed_fc_sgid10, user, 'GRANT', 'AS_IS')
         except:
             print(f'could not update privileges to {renamed_fc_sgid10}')
+
+
+def compare():
+    '''compares data sets between SGID and SGID10 and returns the tables that are different
+    '''
+    dbo_owner = os.path.join(os.getenv('SWAPPER_CONNECTION_FILE_PATH'), 'SGID10', 'SGID10_sde.sde')
+
+    if not os.path.exists(dbo_owner):
```

Comment:
i would recommend replacing all os.path usages with pathlib in any new code you write.

Reply:
Done.